src/parsec.rs

```diff
@@ -387,7 +387,7 @@ fn create(rng: &mut MainRng, full_id: FullId, gen_pfx_info: &GenesisPfxInfo) ->
             &gen_pfx_info.first_info.member_ids().copied().collect(),
             gen_pfx_info.first_state_serialized.clone(),
             ConsensusMode::Single,
-            Box::new(rng::new_from(rng)),
+            Box::new(rng),
```

Comment:
I'm thinking we still want to create a new rng here?
What happen, are we trying to box a mut reference?

Reply:
Ah ye, `OsRng` now (since 0.7) does not return a `Result`. On first use it constructs itself. So the use is a bit simpler now.
I was a bit torn with rand and all it's badly breaking changes with versions. Dalek is now 0.7, threshold_crypt 0.6.5 and parsec is 0.6.5 IIRC. They do not interoperate well at all. Then was thinking why force rand via libs and the only reason is deterministic everything (it's not though). However, I am not sure it matters a node has the exact same name between runs as the name does not matter as long as it fits a section. Same with threshold_crypto (it allows for rand ignoring by having a public random call). Again I am not sure the exact same keyshares matters there, we must assume that crate works. 
So leaves parsec and we can control that, but again do we need to from routing?